package.json

```diff
@@ -53,5 +54,6 @@
   },
   "type": "commonjs",
   "main": "./src/index.js",
-  "typings": "./src/index.d.ts"
+  "typings": "./src/index.d.ts",
+  "packageManager": "yarn@1.22.21+sha1.1959a18351b811cdeedbd484a8f86c3cc3bbaf72"
```

Comment:
why do we have specific packageManager defined? would it break if it is not yarn 1.22.21?

Reply:
I didn't noticed this, I think it was added automatically. Let me check what is this.